### ISSUE TITLE
Image Rotation

### ISSUE THREAD

**Issue by smoki3** [jaredpetersen/raspi-live#34]
Hi,

is it possible to implement a Image Rotation for 90, 180 an 270 degree?

**Comment by jaredpetersen** [comment]
Yes, it looks like raspivid (the camera module CLI) supports rotation:

> `--rotation,    -rot        Set image rotation (0 - 359)`
> Sets the rotation of the image in the viewfinder and resulting image. This can take any value from 0 upwards, but due to hardware constraints only 0, 90, 180, and 270 degree rotations are supported.

Source: https://www.raspberrypi.org/documentation/raspbian/applications/camera.md

This feature could be added to raspi-live as well and would be a good first issue for someone.

**Comment by smoki3** [comment]
Got it working with inserting this option in server.js and cli.js
